src/Features/LanguageServer/Protocol/Features/Diagnostics/EngineV2/DiagnosticIncrementalAnalyzer.cs

```diff
@@ -222,6 +222,8 @@ private static DiagnosticAnalysisResult GetResultOrEmpty(ImmutableDictionary<Dia
         public void LogAnalyzerCountSummary()
             => _telemetry.ReportAndClear(_correlationId);
 
+        public int Priority => 0;
```

Comment:
Probably add a comment that `DiagnosticIncrementalAnalyzer` has the highest priority (lowest int value) amongst all incremental analyzers (others have priority 1).